rbi/sorbet/t.rbi

```diff
@@ -151,7 +151,7 @@ module T::Configuration
   def self.disable_vm_prop_serde; end
   def self.hard_assert_handler(str, extra); end
   def self.hard_assert_handler=(value); end
-  def self.inline_type_error_handler(error); end
+  def self.inline_type_error_handler(error, opts); end
```

Comment:
if there is a default for `opts` in the source, there should be a default in the RBI as well

```suggestion
  def self.inline_type_error_handler(error, opts={}); end
```

Reply:
I did this to keep this consistent with the RBI for `call_validation_error_handler` and `sig_validation_error_handler`, which have an optional `opts` param in the code but not in the RBI since I wasn't sure if that was intentional. I updated the RBI for all three to be `opts={}` to match their runtime args.

Reply:
yep not intentional, thanks for fixing!